lib/puma/cluster.rb

```diff
@@ -171,14 +181,15 @@ def all_workers_idle_timed_out?
       (@workers.map(&:pid) - idle_timed_out_worker_pids).empty?
     end
 
-    def check_workers(refork = false)
+    def check_workers(in_phased_restart)
```

Comment:
phased_restart mechanics are ambiguous today, they can indicate both a complete restart (replace all workers) and a refork (replace _most_ workers except for the template). mold_worker requires differentiating restart from refork, so pass the variable through instead of reducing it to a `refork` boolean.